res/fm/ecl/ecl_run.py

```diff
@@ -282,9 +283,22 @@ def _get_legacy_run_command(self):
                 self.base_name,
             ]
 
+    def _get_log_name(self, eclrun_config=None):
```

Comment:
I see this config is passed all over `EclRun` methods (it doesn't related to your  PR just find it strange that `EclRun` doesn't have one when calling constructor)

Reply:
This has to do with the the two different ways of running eclipse. For newer version ( >= 2018.1) we rely on eclrun to run eclipse (which is much simpler), but for older versions we use our own config. The new way was added on top to not interfere with the existing setup as much as possible, but when all versions < 2018.1 is not supported in Equinor anymore, we can greatly simplify this code!